devel/mps_load_test.sh

```diff
@@ -208,7 +202,8 @@ cd "$src/devel"
 
 # Build a version of mod_pagespeed with all optimizations enabled, but with
 # a build that includes DCHECKs.
-make -j8 CONF=$compile_mode apache_trace_stress_test_server \
```

Comment:
Thanks, that helps a lot with understanding. I'm having a go at killing `GIT_DEBUG_BIN` and `GIT_RELEASE_BIN`... 

One more observation: If you run `loadtest_collect_corpus.sh` on a clean checkout,  apache responds with every request with the stock hello world response (it works!).
If I run `apache_debug_smoke_test` and proceed to  run `loadtest_collect_corpus.sh` I get a real corpus.